bundles/org.openhab.binding.dirigera/src/main/resources/OH-INF/thing/blind.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dirigera"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="blind">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="gateway"/>
+		</supported-bridge-type-refs>
+
+		<label>Blind</label>
+		<description>Window or door blind</description>
+
+		<channels>
+			<channel id="blind-state" typeId="blind-state">
+				<label>Blind State</label>
+				<description>State if blind is moving up, down or stopped</description>
+			</channel>
+			<channel id="current" typeId="dimmer">
+				<label>Current Level</label>
+				<description>Current blind level</description>
+			</channel>
+			<channel id="target" typeId="dimmer">
+				<label>Target Level</label>
+				<description>Target blind level</description>
+			</channel>
```

Comment:
Why not combine these two channels ? 
Usually they are one single channel for blinds, curtains etc.

Reply:
Merged